src/Symfony/Component/EventDispatcher/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * Added autoconfigurable attributes.
```

Comment:
what about this?
Add `EventListener` attribute for declaring listeners on PHP 8

Reply:
Right. The current text is the one that should've gone to the DI component's change log. I'll fix that.